src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/Email.java

```diff
@@ -17,7 +17,7 @@ public final class Email extends AggregateRoot<EmailId> {
     private String content;
     private Set<Attachment> attachments;
 
-    private Email(EmailId id, String content, EmailMetadata emailMetadata,
+    public Email(EmailId id, String content, EmailMetadata emailMetadata,
```

Comment:
Why did you change this? There are factory `create` factory methods

Reply:
oh I didn't noticed 🤣 

great catch!